ush/templates/FV3LAM_wflow.xml

```diff
@@ -105,14 +105,16 @@ tasks; and the "FCST" type is used for the RUN_FCST_TN task.
 
 ]>
 
-<workflow realtime="T" scheduler="&SCHED;" cyclethrottle="3" cyclelifespan="01:00:00:00">
+<workflow realtime="T" scheduler="&SCHED;" cyclethrottle="25" cyclelifespan="01:00:00:00">
 {# Double quotes are required inside the strftime! Expect an error from reading the template if using single quotes. #}
+{%- if (run_task_make_grid or run_task_make_orog or run_task_make_sfc_climo) %}
   <cycledef group="at_start">{{ cdate_first_cycl.strftime("%M %H %d %m %Y *") }}</cycledef>
-  <cycledef group="forecast">{{ cdate_first_cycl.strftime("%Y%m%d%H%M") }} {{ cdate_last_cycl.strftime("%Y%m%d%H%M") }} {{ cycl_freq }}</cycledef>
+{% endif %}
+  <cycledef group="forecast">00 {% for cycl_hr in cycl_hrs[:-1] -%} {{ cycl_hr }},{%- endfor -%}{{ cycl_hrs[-1] }} * * {{ cdate_first_cycl.strftime("%Y") }}-{{ (cdate_last_cycl.strftime("%Y")|int)+1 }} *</cycledef>
 {%- if do_dacycle %}
-  <cycledef group="dacycle">00 03,06,09,15,18,21 {{cdate_first_cycl.strftime("%d") }}-{{ cdate_last_cycl.strftime("%d") }} {{ cdate_first_cycl.strftime("%m") }} {{cdate_first_cycl.strftime("%Y") }} *</cycledef>
+  <cycledef group="dacycle">00 {% for dacycl_hr in dacycl_hrs[:-1] -%} {{ dacycl_hr }},{%- endfor -%}{{ dacycl_hrs[-1] }} * * {{ cdate_first_cycl.strftime("%Y") }}-{{ (cdate_last_cycl.strftime("%Y")|int)+1 }} *</cycledef>
```

Comment:
I'm a bit confused about the indexing in this loop. Why not:

```
{% for dacycl_hr in dacycl_hrs -%} {{ dacycl_hr }},{%- endfor -%}
```

Reply:
Because you can't end with a comma

Reply:
Ahhh. Nice. Thanks!